### ISSUE TITLE
‘struct _gpgme_key’ has no member named ‘fpr’

### ISSUE THREAD

**Issue by Keskebeu** [greenbone/gvm-libs#329]
Linnux Mint 18.1 serena

library build fails with the following:

/home/build-tools/gvm-libs/util/gpgmeutils.c: In function ‘create_all_certificates_trustlist’:
/home/build-tools/gvm-libs/util/gpgmeutils.c:437:63: error: ‘struct _gpgme_key’ has no member named ‘fpr’
       g_string_append_printf (trustlist_content, "%s S\n", key->fpr);
                                                               ^
util/CMakeFiles/gvm_util_shared.dir/build.make:134: recipe for target 'util/CMakeFiles/gvm_util_shared.dir/gpgmeutils.c.o' failed
make[2]: *** [util/CMakeFiles/gvm_util_shared.dir/gpgmeutils.c.o] Error 1
CMakeFiles/Makefile2:140: recipe for target 'util/CMakeFiles/gvm_util_shared.dir/all' failed
make[1]: *** [util/CMakeFiles/gvm_util_shared.dir/all] Error 2
Makefile:160: recipe for target 'all' failed
make: *** [all] Error 2

**Comment by pkleinert** [comment]
Same error on Ubuntu 16.04 caused by gpg library 1.6.0 while version 1.7.0+ seems to be required according to: https://www.gnupg.org/documentation/manuals/gpgme/Key-objects.html

**Comment by Keskebeu** [comment]
@pkleinert thank you for the input. I was able to upgrade the gpg to version 2.1.10 (iwas at 1.4) and it gave me no issues to install. After that I was able to build the library for the openvas 6.0 and I probably can build 11.0 gvm-libs for openvas 7.0. I was trying to build the master branch but it was asking for gvm-libs 20.8 and I was not able to find the library anywhere(unless they are under the gvm-libs master branch which i did not look and tried). But I also found an alternative via a repository I found online. Builds openvas for 8.0 and/or 9.0. So I will close the issue. Take care.
